nextcord/ext/dpy/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+nextcord.ext.dpy
+~~~~~~~~~~~~~~~~~~~~~
+An extension module to replicate parts of Discord.py.
+:copyright: (c) 2022-present tag-epic
+:license: MIT, see LICENSE for more details.
+"""
+
+
+from .slash import describe
```

Comment:
im not *too* sure about the naming of this extension either

Reply:
Should be fixed now!